exp-player/addon/components/exp-player.js

```diff
@@ -181,6 +183,10 @@ able to continue the study.
         },
         closeExitWarning() {
             this.set('hasAttemptedExit', false);
+        },
+        updateFramePage(framePage) {
+            this.set('framePage', framePage);
+            this.sendAction('updateFramePage', framePage);
```

Comment:
For compatibility with existing usages (Lookit in particular) It would be ideal to check if the `updateFramePage` action was passed down or not (and only call this if the handler is available)